src/common/config/process-killer-options.ts

```diff
@@ -0,0 +1,21 @@
+import { isWindows } from "../helpers/operating-system-helpers";
```

Comment:
This function is not available anymore. Please use `getCurrentOperatingSystem()` instead and check against the `OperatingSystem` enum.

Reply:
Okay.